liquibase-service/src/main/resources/db/rdb_modern/routines/012-sp_f_page_case_postprocessing-001.sql

```diff
@@ -212,32 +196,37 @@ BEGIN
             drop table #DIMENSION_KEYS_PAGECASEID
             ;
 
-        select L_INV_ADMINISTRATIVE_INC.PAGE_CASE_UID as PAGE_CASE_UID
+        with LOOKUPCTE as (
```

Comment:
More on a performance question- does using a cte have any impact on the time/performance for this section? Switching to the full lookup table across all the page builder dimensions would pull a lot more data as opposed to just the incremental tables.

Reply:
good point, databases usually optimize and make it work like a join even when we have CTEs. But that being said I'm not 100% sure if that is the case with SQL Server. Let me check if I can get to the query profile and see the difference.

Reply:
In the query plan, the predicate is applied (join/where condition) is applied in the single step. We also have index created on the page_case_uid columns on these tables.